### ISSUE TITLE
Experiment 1: Showing data alignment is most important by controlling/fixing the number of tokens

### ISSUE THREAD

**Issue by brando90** [brando90/data-quality-alignment-sft-af#1]
- reproducing figure 4 but for 7B+ models.
- we have h200s 143GB --> 7B, 13B

---

the CE loss goes down most when you train on data that is most aligned to your task.

Important:
- you should not use the actual benchmark or test to collect data **never**
- you should use dev set (or create set synthetically)
- test set should be untouched

---

Setting
- fixing the number tokens, equal number tokens
- cofounders: maybe other variables we aren't controlling

**Comment by brando90** [comment]
TODO: maybe we make a new git issue, but we need other metrics besides cross entropy.

**Comment by brando90** [comment]
https://arxiv.org/pdf/2410.18194
